internal/network/port-receive.go

```diff
@@ -19,6 +20,62 @@ type incomingPacket struct {
 	buffer  []byte
 }
 
+func handleRTCP(getBufferTransports func(uint32) *TransportPair, buffer []byte) {
+	//decrypted packets can also be compound packets, so we have to nest our reader loop here.
+	compoundPacket := rtcp.NewReader(bytes.NewReader(buffer))
+	for {
+		header, rawrtcp, err := compoundPacket.ReadPacket()
+
+		if err != nil {
+			if err == io.EOF {
```

Comment:
Putting this on the previous line would mean less nesting (easier to read)

Reply:
I get that, but I'd like to leave it, so that in the not-eof case, the error will get printed and we'll return. If i compose the two conditionals, then i'd need a second branch anyway.